src/sass/components.scss

```diff
@@ -0,0 +1,17 @@
+@import 'ngeo/routing/routing.scss';
+@import 'ngeo/floor/floor.scss';
+@import 'ngeo/layertree/common.scss';
+@import 'ngeo/layertree/timeslider.scss';
+@import 'ngeo/profile/profile.scss';
+@import 'ngeo/search/search.scss';
+@import 'ngeo/permalink/share.scss';
+@import 'ngeo/import/import.scss';
+@import 'ngeo/contextualdata/contextualdata.scss';
+@import 'ngeo/dropfile/dropfile.scss';
+@import 'ngeo/print/print.scss';
+@import 'ngeo/query/grid.scss';
+@import 'ngeo/query/window.scss';
+@import 'ngeo/editing/editing.css';
+@import 'ngeo/sass/jquery-ui.scss';
+@import '~mapillary-js/dist/mapillary.css';
```

Comment:
It should be optional, it should probably be in `contribs/gmf/apps/desktop_alt/sass/desktop_alt.scss`